docs/guides/upgrading-v3.md

```diff
@@ -36,12 +36,12 @@ That being said, some plugins need to be updated to be installable with v3. In m
 
 ## Upgrading to v3
 
-First, upgrade to the latest v2 version and make sure that you do not see any deprecation warning.
+First, upgrade to the latest v2 version (`npm install -g serverless@2`) and make sure that you do not see any deprecation warning.
```

Comment:
Will this work after v3 is released?

I realized it wasn't obvious "how to do that"

Reply:
Good point, how would people on the standalone binary be able to update v2?

Reply:
Good point.

With standalone binary we can update to a specific version:

```
curl -o- -L https://slss.io/install | VERSION=2.21.1 bash
```

But is it possible to do so with `serverless upgrade` too? I can't find any docs about that.

Reply:
Yes, it's possible with `serverless upgrade` - in order to upgrade to newest major version you have to provide `--major` flag

Reply:
Theoretically `2` matches any `2.x.x` release, so if e.g. outdated v2 is installed, npm could avoid installation assuming it's already installed, but as I test (at least in the case of global installations), in all cases it attempts to update to latest v2. So it'll work

Reply:
Thanks I updated the text then.